.github/workflows/biome-check.yaml

```
@@ -204,14 +194,4 @@ jobs:
-            }
-
-
-
-      - name: Summary
-        run: |
-          echo "## Biome Check Summary" >> $GITHUB_STEP_SUMMARY
-          echo "- Files checked: ${{ steps.biome-check.outputs.files_checked }}" >> $GITHUB_STEP_SUMMARY
-          echo "- Total issues found: ${{ steps.progress.outputs.remaining_issues }}" >> $GITHUB_STEP_SUMMARY
-          echo "- Errors: ${{ steps.biome-check.outputs.errors }}" >> $GITHUB_STEP_SUMMARY
-          echo "- Warnings: ${{ steps.biome-check.outputs.warnings }}" >> $GITHUB_STEP_SUMMARY
+            }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)